src/neptune/exceptions.py

```diff
@@ -1215,3 +1216,18 @@ def __init__(self, attribute: str, path: str):
         {correct}Need help?{end}-> https://docs.neptune.ai/getting_help/
         """
         super().__init__(message.format(path=path_to_str([attribute, path]), **STYLES))
+
+
+class NeptuneMaxDiskUtilizationExceeded(NeptuneException):
+    def __init__(self, disk_utilization: float, utilization_limit: float):
+        message = """
+    {h1}
+    ----NeptuneMaxDiskUtilizationExceeded----------------------------------------------
+    {end}
+    Current disk utilization ({disk_utilization}) exceeds the limit ({utilization_limit}).
```

Comment:
If these are percentages, maybe it's helpful to show percentage signs?

```suggestion
    Current disk utilization ({disk_utilization}%) exceeds the limit ({utilization_limit}%).
```